pyspark-notebook/Dockerfile

```
@@ -32,10 +30,23 @@ RUN apt-get update --yes && \
 
 # Spark installation
 WORKDIR /tmp
-RUN wget -q "https://archive.apache.org/dist/spark/spark-${APACHE_SPARK_VERSION}/spark-${APACHE_SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}-scala${scala_version}.tgz" && \
+
+RUN if [ -z "$scala_version" ] ; then \
```

```suggestion
RUN if [ -z "${scala_version}" ] ; then \
```

Please, use this type of bash variables.